calculate_average_karthikeyan97.sh

```diff
@@ -16,4 +16,14 @@
 #
 
 JAVA_OPTS="-Xms20480m -Xmx40960m "
-java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_karthikeyan97
+
+if [ -f target/CalculateAverage_karthikeyan97_image ]; then
+    #echo "Picking up existing native image 'target/CalculateAverage_karthikeyan97_image', delete the file to select JVM mode." 1>&2
+    target/CalculateAverage_karthikeyan97_image -Xms20480m -Xmx40960m
```

Comment:
To my understanding, the max. memory is restricted to 32GB, @gunnarmorling? Hence, requesting 40GB of Heap does not work.

Reply:
That's the max heap size, so no guarantee that the app will get that much, depending on the system. The environment has 128G, so it actually may get the full 40 (of course it's an excessively large value).